etc/workers.conf

```diff
@@ -120,4 +120,3 @@ reference-list:
     taglib-gm: 1            # loads tags using a worker (performance thing)
     talklib-gm: 1           # loads comments using a worker (performance thing)
     t-memlimit: 1           # for running certain tests
-    userpic-resize-gm: 1    # resizes userpics
```

Comment:
I think dev sites may still use it? Pretty sure mine is set up that way, although I rarely run the entire worker suite in the course of testing things.